src/nvme/fabrics.c

```diff
@@ -531,7 +531,7 @@ static int __nvmf_add_ctrl(const char *argstr)
 	}
 
 	nvme_msg(LOG_ERR, "Failed to parse ctrl info for \"%s\"\n", argstr);
-	errno = EINVAL;
+	errno = ENVME_CONNECT_PARSE;
```

Comment:
This should be the return value, not the errno, to be consistent with the previous changes.

Reply:
Yeah, I think the function is a bit inconsistent with the error handling.

Reply:
Updated accordingly. Internal function (static) return the error code direct. The exported function map the error code to errno and return -1 in the error case.